src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -213,7 +213,7 @@ package com.videojs.providers{
         }
         
         public function get playing():Boolean{
-            return _isPlaying;
+            return _playbackStarted;
```

Comment:
playbackStarted to me sounds like the video has started playing but may currently be paused or playing. Is that what this is meant to be?